### PR TITLE
[ISSUE #521]Remove unused import

### LINKED ISSUE #521: Remove unused import
image
some unused import in project, need to remove

### PR DESCRIPTION
**Pull request by mxsm**
close #521 

- Remove unused import

### CODE REVIEW

File: examples/restful-ws-quarkus/src/test/java/io/cloudevents/examples/quarkus/UserResourceTest.java [hunk 1]
```diff
@@ -3,8 +3,8 @@
 import io.quarkus.test.junit.QuarkusTest;
 import org.junit.jupiter.api.Test;
 
+
```
Comment (pierDipi) [inline examples/restful-ws-quarkus/src/test/java/io/cloudevents/examples/quarkus/UserResourceTest.java:6]: What's the reason for this empty line? Can we remove it?

**Review by pierDipi — COMMENTED** [review]
Thanks!

File: examples/restful-ws-quarkus/src/test/java/io/cloudevents/examples/quarkus/UserResourceTest.java (hunk 1 above)
Comment (mxsm) [reply examples/restful-ws-quarkus/src/test/java/io/cloudevents/examples/quarkus/UserResourceTest.java:6]: @pierDipi  I will remove it and resbumit this pr later.

Comment (pierDipi) [reply examples/restful-ws-quarkus/src/test/java/io/cloudevents/examples/quarkus/UserResourceTest.java:6]:
```suggestion
```

You can alternatively accept this suggestion

**Comment by mxsm** [comment]
@pierDipi  PTAL~ thanks

**Review by pierDipi — APPROVED** [review]
Thanks again!

**Pull request merged**